src/Order/Bulk.php

```diff
@@ -83,7 +83,10 @@ public function process_order_bulk_actions( $redirect, $doaction, $object_ids )
 			foreach ( $object_ids as $order_id ) {
 				$result           = $this->generate_label_and_notes( $order_id, $_REQUEST );
 				$array_messages[] = $result['message'];
```

Comment:
@Abdalsalaam , what do you think if we also check on `$result['message']` if it has value or not?

Reply:
Done, thank you !